src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -17499,6 +17511,9 @@ void Unit::SetFeared(bool apply)
         if (!caster)
             caster = getAttackerForHelper();
         GetMotionMaster()->MoveFleeing(caster, fearAuras.empty() ? sWorld->getIntConfig(CONFIG_CREATURE_FAMILY_FLEE_DELAY) : 0);             // caster == nullptr processed in MoveFleeing
+
+        if (GetTypeId() == TYPEID_PLAYER)
+            sScriptMgr->AnticheatSetSkipOnePacketForASH(ToPlayer(), true);
```

Comment:
please add brackets even on 1-line `if`/`else`/`for`/etc. without brackets it can easily lead to mistakes, I've seen it happen many times including in AC